src/graphql/ccloud.ts

```diff
@@ -69,16 +69,36 @@ export async function getCCloudResources(): Promise<CCloudEnvironment[]> {
       return;
     }
 
+    // parse Flink Compute Pools first for later association with Kafka clusters
+    let flinkComputePools: CCloudFlinkComputePool[] = [];
+    if (env.flinkComputePools) {
+      const envFlinkComputePools = env.flinkComputePools.map(
+        (pool: any): CCloudFlinkComputePool =>
+          new CCloudFlinkComputePool({
+            ...pool,
+            environmentId: env.id as EnvironmentId,
+            name: pool.display_name,
+            maxCfu: pool.max_cfu,
+          }),
+      );
+      flinkComputePools.push(...envFlinkComputePools);
+    }
+
     // parse Kafka clusters and sort by name
     let kafkaClusters: CCloudKafkaCluster[] = [];
     if (env.kafkaClusters) {
-      const envKafkaClusters = env.kafkaClusters.map(
-        (cluster: any): CCloudKafkaCluster =>
-          CCloudKafkaCluster.create({
-            ...cluster,
-            environmentId: env.id,
-          }),
-      );
+      const envKafkaClusters = env.kafkaClusters.map((cluster: any): CCloudKafkaCluster => {
+        // Associate Flink compute pools with the same provider/region
+        const matchingFlinkPools = flinkComputePools.filter(
```

Comment:
Making this an array instead of bool since I figured the mapping can be useful for other cases in the future?